Dockerfile

```diff
@@ -1,6 +1,7 @@
 # syntax=docker/dockerfile:1.0-experimental
 ARG GO_VERSION
 FROM golang:${GO_VERSION}
+RUN apt update && apt install ca-certificates
```

Comment:
Might be overkill for single package, but `&& rm -rf /var/lib/apt/lists/*` is best practice.

Reply:
I don't think that makes sense here, because this is a throwaway build container, i.e. the image isn't pushed, and only selective assets from the image are copied into other images.